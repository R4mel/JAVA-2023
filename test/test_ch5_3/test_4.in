1
8
s1

9
App
9
Apple
9
Samsung
0
0
0
