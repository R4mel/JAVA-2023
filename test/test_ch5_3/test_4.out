
******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 1
[0] p0 10 70.0 false :Gwangju Nam-gu Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=============== Person Management Menu ================
= 0.exit 1.display 2.clear 3.reset 4.remove           =
= 5.copy 6.append 7.insert 8.login 9.dispStudent(5_3) =
= 10.dispPhone(5_3)                                   =
=======================================================
menu item? 8
user name: s1
password: 

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3)                  +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 9
maker of phone to change(ex: Samsung or Apple)? App
App: WRONG phone's maker

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3)                  +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 9
maker of phone to change(ex: Samsung or Apple)? Apple
s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
s1's Phone: Apple

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3)                  +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 9
maker of phone to change(ex: Samsung or Apple)? Samsung
s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
s1's Phone: SAMSUNG

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3)                  +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 0

=============== Person Management Menu ================
= 0.exit 1.display 2.clear 3.reset 4.remove           =
= 5.copy 6.append 7.insert 8.login 9.dispStudent(5_3) =
= 10.dispPhone(5_3)                                   =
=======================================================
menu item? 0

******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 0

Good bye!!
